bb/src/main/java/com/richardmcdougall/bb/BBService.java

```diff
@@ -1028,12 +1036,14 @@ public void GTFO() {
     public static final int kRemoteAudioTrack = 0x01;
     public static final int kRemoteVideoTrack = 0x02;
     public static final int kRemoteMute = 0x03;
```

Comment:
where does the mute command come from? We don't have a button for that?

Reply:
That one was there already - I simply added to it.